src/java/org/apache/cassandra/metrics/DecayingEstimatedHistogramReservoir.java

```diff
@@ -278,6 +296,48 @@ public static int findIndex(long[] bucketOffsets, long value)
         return value <= bucketOffsets[firstCandidate] ? firstCandidate : firstCandidate + 1;
     }
 
+    private static int findIndexDse(long[] bucketOffsets, long value)
```

Comment:
a note about the fact that this method and the one below was copied from DSE would be nice to have here